hapi/hedera-protobufs/services/auxiliary/tss/tss_encryption_key.proto

```diff
@@ -0,0 +1,46 @@
+/**
+ * # Tss Encryption Key Transaction
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.services.auxiliary.tss;
+
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hedera.hapi.services.auxiliary.tss.legacy";
+// <<<pbj.java_package = "com.hedera.hapi.services.auxiliary.tss">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * A transaction body for sending the public TSS encryption key.
+ */
+message TssEncryptionKeyTransactionBody {
+  /**
+   * The raw bytes of the public TSS encryption key of the node sending the transaction.
```

Comment:
```suggestion
   * An array of public TSS key bytes.<br/>
   * This is the raw bytes of the public TSS encryption key for
   * the node sending the transaction.
```

Reply:
After the decision to keep the transaction bodies in state, is this still relevant? @jsync-swirlds

Reply:
Looks like this file is not in the PR any longer.